internal/namespaces/instance/v1/custom_image.go

```diff
@@ -41,6 +44,7 @@ func imageCreateBuilder(c *core.Command) *core.Command {
 		request := args.CreateImageRequest
 		request.RootVolume = args.SnapshotID
 		request.ExtraVolumes = make(map[string]*instance.VolumeTemplate)
+		request.Organization = args.OrganizationID
```

Comment:
We could also do
```
if args.OrganizationID != "" {
request.Organization = args.OrganizationID
}
```

This would not beak current usage and make generated examples still relevant (until we find a solution)